### ISSUE TITLE
Manage Events: Cleanup

### ISSUE THREAD

**Issue by paaljoachim** [joedolson/my-calendar#91]
Looking at the old method:

![Screen Shot 2021-07-10 at 22 31 44](https://user-images.githubusercontent.com/5323259/125175777-a4d70680-e1ce-11eb-9b59-cdf9588153b0.png)

There is a checkbox next to the ID in the above all the events. 

Looking at the current method:
<img width="860" alt="Screen Shot 2021-07-10 at 22 27 12" src="https://user-images.githubusercontent.com/5323259/125175800-cdf79700-e1ce-11eb-9d3e-d3ec5ed3729d.png">

Here we see a checkbox and a Check all next to the 4 buttons.

The old method feels cleaner.

Adjusting the CSS for the checkboxes next to each ID.

From:
.updates-table td input, .widefat tfoot td input, .widefat th input, .widefat thead td input {
    margin: 0 0 0 8px;
    padding: 0;
    vertical-align: text-top;
}

To:
.updates-table td input, .widefat tfoot td input, .widefat th input, .widefat thead td input {
    margin: 2px 0 0 0; ---> Could be changed to margin-top.
    padding: 0;
    vertical-align: text-top;
}

![Manage-Events-ID-checkbox-adjusting-CSS](https://user-images.githubusercontent.com/5323259/125176050-bde0b700-e1d0-11eb-8b2a-0e5002114026.jpg)

From:
#my-calendar-admin-table th[aria-sort] a:after {
    vertical-align: bottom;
    display: inline-block;
    margin-left: 10px;
    font-family: dashicons;
}

To:
#my-calendar-admin-table th[aria-sort] a:after {
    vertical-align: bottom;
    display: inline-block;
    margin-left: 3px;
    font-family: dashicons;
}

Both adjustments added:
![Manage-Events-Arrow-adjusting-CSS](https://user-images.githubusercontent.com/5323259/125176161-a35b0d80-e1d1-11eb-94ad-7685f711b64a.jpg)

Adding the checkbox next to the top ID area.

![Manage-Events-Checkbox-next-to-ID](https://user-images.githubusercontent.com/5323259/125176264-6e02ef80-e1d2-11eb-9903-6b61c79a183b.jpg)

I am not sure if adding the checkbox to the ID area and removing the Check All is the best idea. I continued to explore a cleanup of the screen.

![Manage-Events-Cleanup](https://user-images.githubusercontent.com/5323259/125176464-dc947d00-e1d3-11eb-91d2-6cafeb3e0290.jpg)

The important part is finding ways to give space for each grouping of element. The mockup shows a way to give space.

**Comment by joedolson** [comment]
Took me a while to figure out what the final mockup is showing; I think that the changes are to move the event status navigation into the upper right & re-align the bulk actions with the search UI; is there anything else there?

I won't be moving the check-all option back into the table; I find that an awkward UI.

**Comment by paaljoachim** [comment]
> Took me a while to figure out what the final mockup is showing; I think that the changes are to move the event status navigation into the upper right & re-align the bulk actions with the search UI; is there anything else there?

In this case I went a bit overboard on the details...
Yeah you got it. I moved the Published - Drafts etc (event status) up to the right.
Aligned the bulk actions with the Search UI. Right now I am not sure where to place the -checkbox- Check all.

Here is one suggestion. I am hesitant about the solution though.
![Manage-events-screen-UI](https://user-images.githubusercontent.com/5323259/125210884-c30e3680-e2a2-11eb-9159-aebb14adc0d9.jpg)
